tests/installation/first_boot.pm

```diff
@@ -29,6 +29,13 @@ sub run() {
             return;
         }
         mouse_hide();
+        if (match_has_tag("dm-nousers")) {
+            #no users created during installation, logging in as root
+            send_key 'tab';    #set focus to "Not listed?"
+            send_key 'ret';
+            wait_idle;
```

Comment:
I don't need this, but for some reason there is wait_idle in if (match_has_tag("sddm")) {...} else {send_key "ret"; wait_idle";}